src/sentry/tasks/post_process.py

```diff
@@ -45,7 +45,7 @@ def _capture_stats(event, is_new):
     if not platform:
         return
     platform = platform.split('-', 1)[0].split('_', 1)[0]
-    tags = {'platform': platform}
+    tags = {'platform': platform, 'use_rust_normalize': event.data['use_rust_normalize']}
```

Comment:
Can we do `data.get`, here, just in case?

Reply:
done